objetos/Aluno.java

```diff
@@ -1,31 +1,43 @@
 package objetos;
 
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Aluno {
+
     private String nome;
     private int idade;
-    public void listarCurso() {
-
-    }
-    public void adicionarCurso() {
-
-    }
-    public void removerCurso() {
 
+    public Aluno(String nome, int idade){
+        this.nome = nome;
+        this.idade = idade;
     }
 
     public String getNome() {
         return nome;
     }
 
-    public void setNome(String nome) {
-        this.nome = nome;
-    }
-
     public int getIdade() {
         return idade;
     }
 
     public void setIdade(int idade) {
+        if (idade < 0 || idade > 120)
+            throw new IllegalArgumentException("Idade deve estar entre 0-120");
         this.idade = idade;
     }
+
+    public void setNome(String nome) {
+        this.nome = nome;
+    }
+
+    @Override
+    public String toString(){
+        System.out.printf("| %13s | %6s |\n", "Nome", "Idade");
```

Comment:
Acredito que o cabeçalho seja melhor mostrar no console quando for exibido a lista de pessoas, ao invés de mostrar para cada nova pessoa ou quando fazer o toString()

- Portanto, acho que podemos descartar a linha 36.